lib/main_common.pm

```diff
@@ -2355,9 +2355,14 @@ sub load_security_tests_selinux {
     # ALWAYS run following tests in sequence because of the dependencies
     # Setup - install SELinux necessary packages
     loadtest "security/selinux/selinux_setup";
-
     loadtest "security/selinux/sestatus";
     loadtest "security/selinux/selinux_smoke";
+
+    # Change SELinux from "permissive" mode to "enforcing" mode for testing
+    loadtest "security/selinux/enforcing_mode_setup";
+
+    # The following test modules must be run after "enforcing_mode_setup"
```

Comment:
1. Does it mean we only test Permissive mode only before for the following modules? 

2. If yes, will we keep the permissive mode testing too?

Reply:
1. yes
2. yes

Reply:
Okay Thanks.